locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/networking/JcloudsLocationSecurityGroupCustomizer.java

```diff
@@ -127,6 +127,7 @@ protected JcloudsLocationSecurityGroupCustomizer(String applicationId) {
     }
 
     protected JcloudsLocationSecurityGroupCustomizer(String applicationId, Supplier<Cidr> sshCidrSupplier) {
+        super();
```

Comment:
This line is redundant - it happens automatically. But I'm fine with leaving it in if you think it makes it clearer (or helps with breakpoints perhaps?)